app/views/course/pick.njk

```diff
@@ -7,7 +7,7 @@
 
 {% block primary %}
   <div class="govuk-form-group">
-    <label for="courses-{{ courseId }}-course" class="govuk-label">Course name and code</label>
+    <label for="courses-{{ courseId }}-course" class="govuk-label">Enter course name and code</label>
```

Comment:
I don't think we should be adding "Enter" here.

Reply:
Spoke to @laurahermionetennant92 about this, we'll keep the label the same but add some help text to describe what a user must do.